domain/src/Entity/Domain.php

```diff
@@ -173,10 +173,9 @@ public static function preCreate(EntityStorageInterface $storage_controller, arr
       'status' => 1,
       'weight' => count($domains) + 1,
       'is_default' => (int) empty($default),
-      // {node_access} still requires a numeric id.
```

Comment:
The `$creator` variable above (`$creator = \Drupal::service('domain.creator');`) is unused now.